src/components/HeaderLink.astro

```diff
@@ -14,20 +14,13 @@ const isActive = href === pathname || href === "/" + subpath?.[0];
   href={href}
   class:list={[
     className,
-    { active: isActive },
-    "text-whitee font-Montserrat text-lg font-normal uppercase hover:text-lightyellow",
+    {
+      "font-bold": isActive,
```

Comment:
this means, that class `font-bold` will be applied when `isActive` is true, in previous setup, there would be `"font-bold underline": isActive` which would make the active link bold with underline